TESTING.md

```diff
@@ -1,70 +1,88 @@
+    jenkins_1    | Please use the following password to proceed to installation:
+    jenkins_1    |
+    jenkins_1    | 70dfe3d2b23841cc96c101c0521716f0
+    jenkins_1    |
+    jenkins_1    | This may also be found at: /var/jenkins_home/secrets/initialAdminPassword
+    jenkins_1    |
+    jenkins_1    | *************************************************************
+    jenkins_1    | *************************************************************
+    jenkins_1    | *************************************************************
+    ``` 
+
+5. Access your Jenkins instance http://localhost:8080
+6. Enter the administrator password in the Getting Started form.
+7. On the next page, click on the "Select plugins to be installed" unless you want to installing suggested plugins. 
+8. Selected desired plugins depending on your needs. You can always add plugins later.
+  - You can skip user creation and stick to the admin user.
+9. Create a user so that you don't have to use the admin credential again (optional)
+10. Continue until the end of the setup process and log back in.
+11. Go to http://localhost:8080/configure to configure the "Datadog Plugin", set your `API Key`.
+  - Click on the "Test Key" to make sure your key is valid
+  - You can set your machine `hostname`
+  - `.*, owner:$1, release_env:$2, optional:Tag3`
+  - You can add `node` optional tag by clicking on the node checkbox
+  - Click on `Advanced...` and set `DogStatsD Host` to `datadog:8125`. This is in order to submit metrics to the dogStatsD Server.
+
+## Create your first job
+
+1. On jenkins Home page, click on "Create a new Job" 
+2. Give it a name and select "freestyle project" (unless you want to do something else).
+3. Then add a build step (execute Shell):
+    ```
+    #!/bin/sh
+    
+    echo "Executing my job script"
+    sleep 5s
+    ```
 
-1. Download the most recent [checkstyle jar](http://sourceforge.net/projects/checkstyle/files/checkstyle/) from SourceForge.
-2. Grab the style checks from [Sun](https://raw.githubusercontent.com/checkstyle/checkstyle/master/src/main/resources/sun_checks.xml) and [Google](https://raw.githubusercontent.com/checkstyle/checkstyle/master/src/main/resources/google_checks.xml).
- * Note: We following 100 character line length, so remove the `<module name="LineLength"/>` line from sun_check.xml.
-3. Run each check, one at a time:
+## Create Logger
+1. Go to http://localhost:8080/log/
+2. Give a name to your logger - For example `datadog`
+3. Add entries for all `org.datadog.jenkins.plugins.datadog.*` packages with log Level `ALL`.
+4. If you now run a job and go back to http://localhost:8080/log/datadog/, you should see your logs
 
-    ```bash
-    java -jar checkstyle-6.13-all.jar -c /sun_checks.xml MyClass.java
-    java -jar checkstyle-6.13-all.jar -c /google_checks.xml MyClass.java
-    ```
+This spins up the Jenkins docker image and auto mount the target folder of this repository (the location where the binary is built)
```

Comment:
Lets move this and the below line into either the `Setup` section or into its own `Making Updates` section.